### DISCUSSION TITLE
TV Show in parts, not in seasons

### DISCUSSION THREAD

**Discussion by jorensanbar** [jellyfin/jellyfin#6633] [Q&A]
I'm having trouble adding information for a TV show.

In the case of "La casa de papel", there are 2 seasons originally, but that is only for the version of Canal Antena3 (Spain only).

When Netflix bought the rights, it published the 2 original seasons, divided and edited for 45 mins, and it did so in parts, for this reason, on Netflix there are 5 parts, and not seasons: https://www.netflix.com/title/80192098

To solve this in TMTB, groups were created, so the series has only 3 seasons, but it does not correspond to the information from Netflix: https://www.themoviedb.org/tv/71446-la-casa-de-papel/episode_groups

The groups were created for these occasions, but in Jellyfin, I can't find a way to tell the system to use the episode groups, and not the information from the original seasons.

**Comment by DFP86** [comment]
the same question.... Athought jellyfin have the option with order by someting, but is now working...   The list just can order by Seasons

**Comment by TheMelmacian** [comment]
If you use thetvdb as primary metadata provider for shows, it's the "Alternate Order": https://thetvdb.com/series/la-casa-de-papel#seasons -> Netflix Order: _Usable as "Alternate Order" in API-connected systems._
For other providers I can't say. In my experience thetvdb is the best provider for alternate orders, mainly because it's clear which order to use in jellyfin.

Edit:
Sadly the current tvdb plugin doesn't support the `alternate` order, because it still uses an older API version.
As soon as the plugin is updated to the latest API version (v4), it should be possible to use the alternate order:
https://github.com/jellyfin/jellyfin-plugin-tvdb/issues/49
https://github.com/jellyfin/jellyfin-plugin-tvdb/pull/93

**Comment by DFP86** [comment]
I used the TMBD...
Their webmaster is very arrogant...
They only reserve one TV show collection and delete all of the same collections with so many people in used. （In used the collection is complete, perfect. The reserve one information is broken.）

Many people propose to restore the old collection project. 
They said no.
They just tell you can use their API so that you can order by custom options, just like “[Order by author Nisio Isin (故事线)](https://www.themoviedb.org/tv/46195/episode_group/6224ccddfe077a0043a57f5c)”...

When people tell then jellyfin cant usd this API. They said: Please contact the software developer. （Almost Just say: It no my bussinss. ）

Hehe... They can delete the same collection when the most softwares supports it. But they are not.
so sad... 

Jellyfin! Come on!!!~~~~ （Not urging, just tribute to the developerss）
